src/components/FineForm/component.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC, useEffect, useState } from 'react';
```

Comment:
I want to suggest you use another flow for naming component
example
The component has the name FineForm
We create a folder FineForm, index.ts for exports/imports, and the component should have the name FineForm.tsx

Each component will have own name. If needs to replace logic in custom hooks only for this component, lets name FineForm.hooks.ts